nms/app/packages/magmalte/grafana/dashboards/Dashboards.js

```diff
@@ -306,6 +322,70 @@ export const GatewayDBData: GrafanaDBData = {
   ],
 };
 
+export const SubscriberDBData: GrafanaDBData = {
+  title: 'Subscribers',
+  description:
+    'Metrics relevant to subscribers. Do not edit: edits will be overwritten. Save this dashboard under another name to copy and edit.',
+  templates: [networkTemplate, imsiTemplate, apnTemplate],
+  rows: [
+    {
+      title: '',
+      panels: [
+        {
+          title: 'UE Data Usage In',
+          targets: [
+            {
+              expr:
+                'sum(ue_reported_usage{IMSI=~"$imsi", direction="down"}) by (IMSI, apn, msisdn)',
+              legendFormat: '{{IMSI}}, MSISDN: {{msisdn}}, APN: {{apn}}',
+            },
+          ],
+          unit: 'decbytes',
+          description: 'Inbound data per subscriber measured in bytes.',
+        },
+        {
+          title: 'UE Data Usage Out',
+          targets: [
+            {
+              expr:
+                'sum(ue_reported_usage{IMSI=~"$imsi", direction="up"}) by (IMSI, apn, msisdn)',
```

Comment:
How do we deal with the problem where a service goes down and resets to 0 ? @Scott8440 might have more to add here. In subscriber dashboard in FWA we use ue_traffic metric and we display it based on the increase of bytes used in a certain bucket, for e.g. subscriber used 1MB in last 1 hour. As a sidenote, should we change the subscriber dashboard in NMS to use ue_reported_usage instead of ue_traffic ?

Reply:
Yeah we show raw counters for several graphs right now and just accept that service restarts will cause restarts to the counter. That's why we also have the rate() graph which handles for service restarts. I think it's ok to have the raw counter as well

Reply:
@karthiksubraveti We should use ue_reported_usage to keep a more consistent reporting between NMS dashboards, plus as this metric aligns more with the actual data usage reported on the session / flows rather than relying on monitoring on the policies.

Reply:
@ardzoht Would it be possible to make this change in https://github.com/magma/magma/blob/master/nms/app/packages/magmalte/app/views/subscriber/SubscriberChart.js#L73 as well. Thanks

Reply:
@karthiksubraveti Updated the SubscriberChart